src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -79,10 +79,10 @@ private void InitializeArgs(NativeCompileSettings config)
             // Add Stubs
             argsList.Add(Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04/lxstubs.cpp"));
```

Comment:
I will do this as part of fixing https://github.com/dotnet/corert/issues/309 that will move a bunch of headers from AppDep into ILCompiler.SDK. I will use that oppurtunity to merge the code into bootstrapper.

This change is simply to expose the --ilcsdkpath support.